docs/source/getting_started/benchmarks/imagenette_benchmark.py

```diff
@@ -870,8 +869,8 @@ def __init__(self, dataloader_kNN, num_classes):
         utils.deactivate_requires_grad(self.projection_head_momentum)
 
         # smog
-        self.n_groups = 300 # 2% malus vs optimal setting of 3000 groups
-        memory_bank_size = 300 * batch_size # because we reset the group features every 300 iterations
+        self.n_groups = 300
+        memory_bank_size = 10000
```

Comment:
Why is this no longer 300 * batch_size ?

Reply:
Because `300 * batch_size >> n_samples`